test/api.js

```diff
@@ -124,6 +124,34 @@ describe('api', function () {
 		});
 	});
 
+
+	//unit tests of user
+	describe('/:id/data', function () {
+		it('should be able to get data from user', function() {
+			return client
+			.get('/:id/data')
+			.then((data) => {
+				expect(data).to.be(Object);
+				should.exist(data.result);
+
+			})
+		});
+
+
+		// meeh..
+		it('should be able to post data if user is self or admin for group that self belongs to', function() {
```

Comment:
Hold de enkelte scenarier simple. Måske lave 3 tests:

- should be able to post data to myself.
- should be able to post data to a member of a group I am admin of.
- should not be able to post data to a member outside my control.